### DISCUSSION TITLE
e107 theme development - tutorial

### DISCUSSION THREAD

**Discussion by Jimmi08** [e107inc/e107#5332] [Ideas]
Hi,
for my other project, I (our company) will pay for special software for online courses.  At least for the year.

Will there be any interest in theme development curses? 

For free - only that admin dashboard will be in Slovak, but I can create a special category for e107 stuff.  

For the English dashboard, I need to pay the next 50 euros/per month and it is too much for me.

I just started on the new theme and I have so many tricks or solutions already and nobody knows about it.    

The main plus is a private discussion for each lesson...

From the beginning a bit chaotic...   for example - I now do templates for members-only sites and I can show this topic now. But lately, I can organize it better, when I have time. 

Let me know.

**Comment by tgtje** [comment]
Very much appreciated. For _me_ due to circumstances not an option
